app/src/main/kotlin/com/wire/android/ui/calling/SharedCallingViewModel.kt

```diff
@@ -211,8 +211,9 @@ class SharedCallingViewModel @Inject constructor(
         sharedFlow.first()?.let { call ->
             callState = callState.copy(
                 callStatus = call.status,
-                callerName = call.callerName
-            )
+                callerName = call.callerName,
+                isCbrEnabled = call.isCbrEnabled
+                )
```

Comment:
```suggestion
                callerName = call.callerName,
                isCbrEnabled = call.isCbrEnabled
            )
```

to make them properly inline hehe